rootio/scanner.go

```diff
@@ -54,11 +54,36 @@ func (s *baseScanner) SeekEntry(i int64) error {
 	if s.err != nil {
 		return s.err
 	}
+	if s.chain {
+		if s.cur >= s.n {
+			itree := s.findTree(i)
+			s.loadTree(itree)
+		}
+	}
 	s.i = i
 	s.cur = i - 1
 	return s.err
 }
 
+// find the tree number in the chain that contains entry i
```

Comment:
the usual doc line would be more like:
```go
// findTree finds the tree ...
```

also: could you document what happens when we don't find the tree?
(w/o looking at the method, I would expect the returned value to be, say, `-1`.)